src/setupTests.js

```diff
@@ -9,3 +9,12 @@ import '@testing-library/jest-dom';
     value: 500,
   });
 });
+
+// Silence web workers warning from `drei`
+const { log } = console;
+// eslint-disable-next-line no-console
+console.log = (...args) => {
+  if (args.length === 0 || !/web workers not allowed/.test(args[0])) {
+    log(...args);
+  }
+};
```

Comment:
What does the warning says ?

Reply:
I managed to find a better solution: I just mock the problematic package (`troika`), which doesn't get used anyway.